auth/auth.example.yml

```diff
@@ -3,3 +3,9 @@
   # WILL CAUSE ERRORS WHEN TRYING TO BUY/SELL
   gateio_api: "GATEIO_APIv4_KEY"
   gateio_secret: "GATEIO_SECRET_v4"
+
+  # Getting a bot token: https://core.telegram.org/bots#6-botfather
```

Comment:
I just thought I might try this myself. It only worked on mobile for me even though I use Telegram on my PC. Maybe it's supposed to be that way but then a comment here might be nice.

Reply:
It's propably because the link to the botfather [https://t.me/botfather](https://t.me/botfather) does not work if you use telegram in the browser only.  

The question is how far do we go in documenting this?  
Another way would be just putting it as a prerequisite, e.g.  
```
For setting up telegram notifications you need a bot token and a chat-id.
Read more about it here: https://core.telegram.org/bots#6-botfather
```

or another idea:

----

How to get your bot data:
1. Get a telegram bot by talking to the botfather [https://t.me/botfather](https://t.me/botfather).  
If this link doesn't work, search for `@BotFather` in Telegram and type `/start`
2. After creating your new bot, search for it on telegram and send some messages to it.
3. Go visit `https://api.telegram.org/bot<YOUR_TOKEN>/getUpdates` and find the chat-id of the conversation between you and your bot

Reply:
Turns out, it worked after I waited some while and send more messages. Weird but alright. :)
I think your description is enough, people just need to be patient. :D

Reply:
I mean, if you want to add a hint that most should be done on the phone and people should be fine. Then again, it's primarily a phone app so it should be fine.